docs/Stamp.md

```diff
@@ -0,0 +1,25 @@
+# Stamp
+
+Use the `<Stamp> </Stamp>` component to render a primitive text Stamp.
```

Comment:
Might be good to clarify where this component could be used, or what the rationale for it is